package-lock.json

```diff
@@ -1,53 +1,46 @@
 {
   "name": "@okta/okta-signin-widget",
-  "version": "1.13.0",
+  "version": "2.2.0",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
     "@okta/okta-auth-js": {
       "version": "1.8.0",
-      "resolved": "https://registry.npmjs.org/@okta/okta-auth-js/-/okta-auth-js-1.8.0.tgz",
-      "integrity": "sha1-D9c2XyHIcznV4ko2tP4FJDC3xyg=",
+      "resolved": "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta/@okta/okta-auth-js/-/@okta/okta-auth-js-1.8.0.tgz",
```

Comment:
I think that this will be a problem - these point to our internal registry. To fix this, rename your ~/.npmrc file and then:

1. `npm cache clear`
2. `rm -rf node_modules`
3. `npm install`